config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_vertexaiindexes.vertexai.cnrm.cloud.google.com.yaml

```diff
@@ -222,60 +222,317 @@ spec:
                       type: string
                   type: object
                 type: array
-              createTime:
-                description: The timestamp of when the Index was created in RFC3339
-                  UTC "Zulu" format, with nanosecond resolution and up to nine fractional
-                  digits.
+              observedGeneration:
+                description: ObservedGeneration is the generation of the resource
+                  that was most recently observed by the Config Connector controller.
+                  If this is equal to metadata.generation, then that means that the
+                  current reported status reflects the most recent desired state of
+                  the resource.
+                type: integer
+              observedState:
+                description: The observed state of the underlying GCP resource.
+                properties:
+                  createTime:
+                    description: The timestamp of when the Index was created in RFC3339
+                      UTC "Zulu" format, with nanosecond resolution and up to nine
+                      fractional digits.
+                    type: string
+                  deployedIndexes:
+                    description: The pointers to DeployedIndexes created from this
+                      Index. An Index can be only deleted if all its DeployedIndexes
+                      had been undeployed first.
+                    items:
+                      properties:
+                        deployedIndexId:
+                          description: The ID of the DeployedIndex in the above IndexEndpoint.
+                          type: string
+                        indexEndpoint:
+                          description: A resource name of the IndexEndpoint.
+                          type: string
+                      type: object
+                    type: array
+                  etag:
```

Comment:
Nit: This field is super-volatile and not very useful to KCC users.  We should consider whether we want to exclude etag for status always (with a rule, to make it easy!).  We might also apply the same argument to createTime (and updateTime), but here createTime was pre-existing.

I'm inclined to say "let's do this, but think carefully about the next etag based on what we learn here"